test/models/concerns/audit_extensions_test.rb

```diff
@@ -35,4 +35,10 @@ def setup
     a = Audit.where(auditable_type: 'Setting')
     assert_equal "[encrypted]", a.last.audited_changes["value"][1]
   end
+
+  test "search for type=lookupvalue in audit" do
+    key   = VariableLookupKey.create!(:key => "ntp", :path => "domain", :puppetclass => Puppetclass.first)
+    LookupValue.create!(:value => "ntp.mydomain.net", :match => "domain =  mydomain.net", :lookup_key => key)
+    assert Audit.search_for("type = lookupvalue").count > 0
```

Comment:
Please try to avoid generic `asserts` or specify a custom message. I'd probably use `refute_empty` here